### ISSUE TITLE
Not working on windows

### ISSUE THREAD

**Issue by Busyrev** [fual/nagibabel.js#6]
OS: Windows 10
npm: 5.6.0
node:v9.4.0

command tp install: npm install https://github.com/fual/nagibabel.js
output:
```cmd
> nagibabel.js@1.0.0 postinstall E:\qq\ww\node_modules\nagibabel.js
> sh ./install.sh

'sh' is not recognized as an internal or external command,
operable program or batch file.
npm WARN nagi-test@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nagibabel.js@1.0.0 postinstall: `sh ./install.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nagibabel.js@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\m.busyrev\AppData\Roaming\npm-cache\_logs\2018-03-21T09_04_25_586Z-debug.log
```

I would like to see powershell support :)

**Comment by underoot** [comment]
I need this!

**Comment by rmbaad** [comment]
Same problem

**Comment by underoot** [comment]
Where my windows support, bro?

**Comment by MrKashlakov** [comment]
On my windows machine not working!

**Comment by eduardnikolenko** [comment]
Fixed in #7

**Comment by grachpower** [comment]
same problem
